lib/portage/util/cpuinfo.py

```diff
@@ -48,3 +50,22 @@ def get_cpu_count():
         return multiprocessing.cpu_count()
     except (ImportError, NotImplementedError):
         return None
+
+
+def makeopts_to_job_count(makeopts):
+    """
+    Parse the job count (-jN) from MAKEOPTS. Python version of
+    bin/isolated-functions.sh's ___makeopts_jobs().
+
+    @return: Number of jobs to run or number of CPUs if none set.
+    """
+    if not makeopts:
+        return get_cpu_count()
+
+    # TODO: Fix
+    jobs = re.match(".*(j|--jobs=\s)\s*([0-9]+)", makeopts)
+
+    if not jobs or not jobs.groups()[1]:
```

Comment:
I don't think the group can be empty, can it?

Reply:
Let me check again, I think I had it somehow before, but it may have been with an inferior regex I was using at first.